frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -157,6 +164,43 @@ const Settings: NextPage = () => {
                     {labelCollectionWarning}
                   </div>
                 </div>
+                <div className={styles.field}>
+                  <h2 className={styles["field-heading"]}>
+                    {l10n.getString("setting-label-api-key")}
+                  </h2>
+                  <div
+                    className={`${styles["copy-api-key-input"]} ${styles["field-content"]}`}
+                  >
+                    {/* TODO: Convert the following element to <output> when "hidden" */}
+                    <div className={styles["copy-api-key-display"]}>
+                      {profile.api_token}
+                    </div>
+                    <span className={styles["copy-controls"]}>
+                      <span className={styles["copy-button-wrapper"]}>
+                        <button
+                          className={styles["copy-button"]}
+                          title={l10n.getString("settings-button-copy")}
+                          aria-label={l10n.getString("settings-button-copy")}
+                          onClick={copyApiKeyToClipboard}
+                        >
+                          <img
+                            src={copyIcon.src}
+                            alt=""
+                            className={styles["copy-icon"]}
+                          />
+                        </button>
+                        {/* <span
+                          aria-hidden={!justCopied}
+                          className={`${styles["copied-confirmation"]} ${
+                            justCopied ? styles["is-shown"] : ""
+                          }`}
+                        >
+                          {l10n.getString("profile-label-copied")}
+                        </span> */}
```

Comment:
question: is it intentional to leave this commented out?

Reply:
I think that was in-progress work to display a confirmation after copying. I just finished that in 84db24ffdcb55ffa209dc79b54d0495524d49970.